firefox_ui_tests/functional/awesomebar/test_star_in_autocomplete.py

```diff
@@ -0,0 +1,76 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestStarInAutocomplete(FirefoxTestCase):
+    """ This replaces
+    http://hg.mozilla.org/qa/mozmill-tests/file/default/firefox/tests/functional/testAwesomeBar/testSuggestBookmarks.js
+    Check a star appears in autocomplete list for a bookmarked page.
+    """
+
+    PREF_LOCATION_BAR_SUGGEST = 'browser.urlbar.default.behavior'
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.clear_history()
+
+        # Location bar suggests 'History and Bookmarks'
+        self.prefs.set_pref(self.PREF_LOCATION_BAR_SUGGEST, 0)
+
+    def tearDown(self):
+        FirefoxTestCase.tearDown(self)
+
+        self.prefs.reset_pref(self.PREF_LOCATION_BAR_SUGGEST)
+
+    def clear_history(self):
```

Comment:
It would be good to mark it with a TODO given that this has to be part of the places library. It will make it hard for us to update the tests later and to find all those entries.